.changeset/fix-tooltip-replace-popper-with-floating-ui.md

```diff
@@ -0,0 +1,5 @@
+---
+'react-magma-dom': patch
+---
+
+fix(Tooltip): Replace Popper with Floating UI.
```

Comment:
Maybe we should make a note here that the default tooltip `arrowSize` has changed in case adopters are using a custom size in their theme.